studio/src/pages/[organizationSlug]/checks-config.tsx

```diff
@@ -0,0 +1,58 @@
+import { NamespaceSelector } from "@/components/dashboard/NamespaceSelector";
```

Comment:
Should we add this config to the lint policy page itself and rename it to Check Configurations?

Reply:
I have mixed feelings for this. Leaning more towards isolating it for checks. You can once discuss with dustin.